src/api/test/unit/webui/webui_helper_test.rb

```diff
@@ -55,4 +55,14 @@ def test_escape_project_list_escaped_forbidden_chars
     output = "['&lt;p&gt;home:Iggy&lt;/p&gt;', '&lt;p&gt;This is a paragraph&lt;/p&gt;']"
     assert escape_nested_list(input), output
   end
+
+  def test_format_projectname
```

Comment:
Why is it moved from 
``
 src/api/test/unit/webui_helper_test.rb 
`` 
to 
``
 src/api/test/unit/webui_helper_test.rb 
``
?